Makefile

```diff
@@ -2,6 +2,7 @@
 BASE_IMAGE_NAME = postgresql
 VERSIONS = 9.4 9.5 9.6
 OPENSHIFT_NAMESPACES = 9.2
+export VERSIONS
```

Comment:
I am a bit scared exporting VERSIONS might break something in the existing scripts. It would be better if you just kept the versions variable local for the `gen-dirs.sh` script. So something like:

```
VERSIONS="$(VERSIONS)" ./gen-dirs.sh
```